apps/microsoft-teams/frontend/src/components/config/NotificationViewMode/NotificationViewMode.tsx

```diff
@@ -11,13 +11,9 @@ import {
 } from '@contentful/f36-components';
 import { MoreHorizontalIcon } from '@contentful/f36-icons';
 import { styles } from './NotificationViewMode.styles';
-import { getContentTypeName, getChannelName } from '@helpers/configHelpers';
-import { Notification, TeamsChannel } from '@customTypes/configPage';
-import {
-  channelSelection,
-  contentTypeSelection,
-  notificationsSection,
-} from '@constants/configCopy';
+import { getContentTypeName } from '@helpers/configHelpers';
+import { Notification } from '@customTypes/configPage';
+import { contentTypeSelection, notificationsSection } from '@constants/configCopy';
 // TODO: update this when we start fetching channel installations
```

Comment:
I might have missed this, is the TODO here still necessary or can we 🧹 ?

Reply:
Good catch! Removed it